packages/job-components/src/register-apis.ts

```diff
@@ -105,10 +107,12 @@ export function registerApis(
         },
     });
 
-    const assetDir = context.sysconfig.teraslice.assets_directory;
+    const configAssetsDir = context.sysconfig.teraslice.assets_directory || [];
+    const assetDir = Array.isArray(configAssetsDir) ? configAssetsDir : [configAssetsDir];
+
     context.apis.registerAPI('assets', {
         getPath(name: string): Promise<string> {
-            return getAssetPath(assetDir || '', assetIds || job.assets, name);
+            return getAssetPath(assetDir || [], assetIds || job.assets, name);
```

Comment:
We now this will be an array
```suggestion
            return getAssetPath(assetDir, assetIds || job.assets, name);
```